### ISSUE TITLE
Unable to batch import

### ISSUE THREAD

**Issue by ssbarnea** [Komet/MediaElch#711]
It seems that the import functionality cannot be used to import all detected movies and the user needs to perform lots of repetive steps fro importing each movie.

It should be able to:
* select all importable iterms and perform import
* auto-close the import finished window

**Comment by bugwelle** [comment]
Agreed. Will have to test this as well but your second point is worth a "feature request" flag :smiley: 
Thanks for opening this issue!


Regards,
Andre
